### PR TITLE
chore: hermione -> testplane

### PR DESCRIPTION
**Pull request by KuznetsovRoman**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "hermione-retry-progressive",
-  "version": "0.0.4",
-  "description": "Hermione plugin for adding extra retries at tests with errors matched by patterns.",
+  "name": "@testplane/retry-progressive",
+  "version": "0.0.1",
```
Comment (sipayRT) [inline package.json:3]: `1.0.0`

File: package.json [hunk 2]
```diff
@@ -12,19 +12,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+ssh://git@github.com/gemini-testing/hermione-retry-progressive.git"
+    "url": "git+ssh://git@github.com/gemini-testing/testplane-retry-progressive.git"
   },
   "keywords": [
+    "testplane",
     "hermione",
```
Comment (sipayRT) [inline package.json:19]: did you decide to leave it on purpose? Maybe it makes sense

**Review by sipayRT — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (KuznetsovRoman) [reply package.json:3]: You want to release all of the packages as 1.0.0? (Including testplane itself) I thought about releasing them as 0.1.0

File: package.json (hunk 2 above)
Comment (KuznetsovRoman) [reply package.json:19]: > did you decide to leave it on purpose?

Yes, because the code base is the same.

**Pull request merged**